GitHub-Test-4-PyNLPDiscuss/gak-GitTest4.md

```diff
@@ -2,9 +2,12 @@
 ## I like the idea of finding "hidden themes" within text.
 ### One concern about it that the book addresses is that we could potentially misinterpret certain words and patterns and make false inferences about the text as a result.
 #### I suppose that makes topic modeling somewhat subjective.
+
 ```
 <p> It might be interesting to perform topic modeling on a movie or video game script for a project. 
```

Comment:
TSA: I had a similar idea that involved using NLP over scripts. I think it could be very useful to use to help understand intricacies of literary devices or other important information!